doc/markup/code.rst

```diff
@@ -182,13 +182,24 @@ Includes
    ``start-after`` is given as a string option, only lines that follow the first
    line containing that string are included.  If ``end-before`` is given as a
    string option, only lines that precede the first lines containing that string
-   are included. If used in combination with ``lines``, make sure the latter
-   allows the lines (whose count start at ``1`` at top of file) containing the
-   looked-for strings.
```

Comment:
I do not add a versionchanged string because as previous situation was dysfunctional it only complicates things to document it, the change will be registered in CHANGES if PR is mergd